<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Generar Valores</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Grafo</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>200</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>Insertar ciudades</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_Nombre"/>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_Costo"/>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_NumeroEstrellas"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_AgregarCiudad">
                <property name="text">
                 <string>Agregar</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_4">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="2" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Tiempo</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="spbWeight"/>
              </item>
              <item row="4" column="0" colspan="2">
               <widget class="QPushButton" name="btnAddArista">
                <property name="text">
                 <string>Agregar</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QFrame" name="frame_2">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QComboBox" name="cbFrom">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>10</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="cbOption">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>10</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="cbTo">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>10</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QLabel" name="label_5">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Agregar carretera</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="frame_7">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QFrame" name="frame_5">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QFrame" name="frame_10">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QGroupBox" name="groupBox">
                   <property name="title">
                    <string>Opciones</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QRadioButton" name="btnhideW">
                      <property name="text">
                       <string>Ocultar Peso</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="btnShowW">
                      <property name="text">
                       <string>Mostrar Peso</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_6">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QPushButton" name="pushButton_2">
                   <property name="text">
                    <string>Dijkstra</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton">
                   <property name="text">
                    <string>Árbol de expansión mínima (Prim)</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGraphicsView" name="graphicsView"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QFrame" name="frame_9">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QFrame" name="frame_8">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Nombre:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="lblstars">
                <property name="text">
                 <string>sdsdsds</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Número de Estrellas:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="lblcost">
                <property name="text">
                 <string>sdsdsd</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="lblname">
                <property name="text">
                 <string>sdsd</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Costo:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Datos de la ciudad seleccionada</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTabWidget" name="tabWidget_2">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_3">
              <attribute name="title">
               <string>Matriz de adyacencia</string>
              </attribute>
             </widget>
             <widget class="QWidget" name="tab_4">
              <attribute name="title">
               <string>Tab 2</string>
              </attribute>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
